### ISSUE TITLE
[REQ] \e\ (deGoogled-LOS) build

### ISSUE THREAD

**Issue by MarcoRavich** [Huawei-Dev/android_kernel_huawei_vns#1]
Someone asked a build for _venus_ @ [\e\ community forum](https://community.e.foundation/t/huawei-p9-lite-please/11850): can you help ?

Thanks in advance !

**Comment by Huawei-Dev** [comment]
Yes check xda , LOS 21 Android 14 it's already released for P9 Lite ( Venus )

**Comment by MarcoRavich** [comment]
Yes, I know, I've also reported it in the linked 3ad...
